packages/openapi-ts/src/types/config.ts

```diff
@@ -169,6 +169,16 @@ export interface ClientConfig {
          */
         name?: 'PascalCase' | 'preserve';
       };
+
+  /**
+   * Generate transforms for types?
+   */
+  transform?: {
```

Comment:
What other transforms do you think we'd want in the future? Thinking whether this shouldn't live inside types directly

Reply:
Just didn't want to mix transforms into types right out the gate. Might be folk want something like BigInt etc in the future as was discussed recently, and maybe more configuration to instead use luxon/date-fns etc (though I personally think that's unnecessary heavy lifting)

I guess we could rather make this a visitor callback to match and provide a template fragment for ultimate flexibility if those types of wishes are important

Reply:
Have you considered making `responseTransformer` an array of transforms? How would you stack multiple transforms?

Reply:
Since the transforms are for the model as a whole, the generator will produce all the mutations into the same function. No need to iterate over a bunch of transforms as it's raw JS

I'll do a slightly more comprehensive example around this including arrays and such, should make it clearer :)

Reply:
I've made the tests/index.spec.ts produce a new set of snapshots now, specifically 1 for each client with a dedicated v3-transforms.json which only covers transforms. All implemented across the generated clients and pushed up

Reply:
I see you're on fire today 🚀

Reply:
It's been a productive one!

The new package clients were pretty easy to add support to, so they're included now.

I think the major meat of this PR is ready, just some details left but I'll write separate comments on those lines